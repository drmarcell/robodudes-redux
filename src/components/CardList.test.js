import React from 'react';
import CardList from './CardList';
import { create } from "react-test-renderer";

const mockRobots = [
    {
        key: 'k1',
        id: 1,
        name: 'John',
        email: 'john@gmail.com',

    },
    {
        key: 'k2',
        id: 2,
        name: 'Sally',
        email: 'sally@gmail.com',

    },

]

const cardList = create(<CardList robots={mockRobots} />);

it('expect to render CardList component', () => {
    expect(cardList.toJSON(<CardList robots={mockRobots} />)).toMatchSnapshot();
})