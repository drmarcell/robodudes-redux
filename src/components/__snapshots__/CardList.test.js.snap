// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render CardList component 1`] = `
<div>
  <div
    className="tc bg-washed-green dib br3 pa3 ma2 grow bw2 shadow-5"
  >
    <img
      alt="robots"
      src="https://robohash.org/1?size=200x200"
    />
    <h2>
      John
    </h2>
    <p>
      john@gmail.com
    </p>
  </div>
  <div
    className="tc bg-washed-green dib br3 pa3 ma2 grow bw2 shadow-5"
  >
    <img
      alt="robots"
      src="https://robohash.org/2?size=200x200"
    />
    <h2>
      Sally
    </h2>
    <p>
      sally@gmail.com
    </p>
  </div>
  ;
</div>
`;
